en/docs/security-announcements/cve-justifications/2026/CVE-2023-49735.md

```
@@ -0,0 +1,32 @@
+- WSO2 Identity Server : 7.3.0
+
+### WSO2 JUSTIFICATION
+
+Apache Tiles[^2] 3.x is end-of-life with no security patches available and hence does not have a fix. Due to this reason, we are publishing this CVE justification with a detailed analysis of the CVE, outlining how associated risks are mitigated in WSO2 products and the actions WSO2 is taking in response.
+
+User input is not permitted to set the DefaultLocaleResolver.LOCALE_KEY session attribute, and there are no usages of LOCALE_KEY within the WSO2 codebase. Furthermore, Tiles context processing is not performed explicitly within the application, and Tiles definitions are not locale-dependent. Additionally, XML parsers are configured with XXE protections enabled.
+
+Based on this evidence, we conclude that this vulnerability does not pose a security risk to the impacted versions of WSO2 Products listed above.
+
+### REFERENCES
+[^1]: [https://nvd.nist.gov/vuln/detail/CVE-2023-49735](https://nvd.nist.gov/vuln/detail/CVE-2023-49735)
```

markdownlint-cli2 (0.22.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "^1"

(MD053, link-image-reference-definitions)